apps/test-ui/src/AppDescriptor.ts

```
@@ -3,10 +3,10 @@
 // =============================================================================
 
 import React from 'react';
-import type { AppDescriptor } from 'shell-ui';
+import type { AppDescriptor } from 'shell';
```

ESLint

[error] 'shell' import is restricted from being used. Renamed: import from 'shell'.

(no-restricted-imports)